src/screens/login/index.tsx

```diff
@@ -36,10 +37,14 @@ const Login = ({navigation}: TNavigation<Routes.LOG_IN>) => {
     setFocus('email');
   }, [setFocus]);
 
-  const onSubmit = (data: ILogin) => {
-    signIn(data.email, data.password).then(() => {
+  const onSubmit = async (data: ILogin) => {
+    try {
+      setIsLoading(true);
+      await signIn(data.email, data.password);
       reset();
-    });
+    } catch {
+      setIsLoading(false);
+    }
```

Comment:
To have a minimum visibility on logIn error, and always resetting the loading state, we could do something like,
```suggestion
    } catch(error) {
      console.log('Error on signIn onSubmit -',error);
    } finally {
      setIsLoading(false);
     }
```

Reply:
true